cmd/soroban-cli/src/commands/contract/invoke.rs

```diff
@@ -163,7 +163,7 @@ impl Cmd {
             .transpose()
     }
 
-    fn send(&self, sim_res: &SimulateTransactionResponse) -> Result<ShouldSend, Error> {
+    fn should_send_tx(&self, sim_res: &SimulateTransactionResponse) -> Result<ShouldSend, Error> {
```

Comment:
I don't love the names of these fns, but wanted to get this PR up so I can get some additional eyes on this approach

Reply:
Yeah something like should_send_after_sim? names are hard!